fdb-relational-grpc/src/test/java/com/apple/foundationdb/relational/jdbc/ResultSetContinuationTest.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * ResultSetContinuationTest.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2021-2024 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.relational.jdbc;
+
+import com.apple.foundationdb.relational.api.Continuation;
+import com.apple.foundationdb.relational.api.RelationalResultSet;
+import com.apple.foundationdb.relational.jdbc.grpc.v1.ResultSet;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.sql.SQLException;
+import java.sql.Types;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class ResultSetContinuationTest {
+
+    public static Stream<Arguments> continuationSource() {
+        return Stream.of(
+                Arguments.of(MockContinuation.BEGIN),
+                Arguments.of(MockContinuation.END),
+                Arguments.of(new MockContinuation(Continuation.Reason.TRANSACTION_LIMIT_REACHED, new byte[]{1}, false, false)),
+                Arguments.of(new MockContinuation(Continuation.Reason.CURSOR_AFTER_LAST, new byte[]{1}, false, true)),
```

Comment:
I feel like theses should be different
```suggestion
                Arguments.of(new MockContinuation(Continuation.Reason.TRANSACTION_LIMIT_REACHED, new byte[]{1}, false, false)),
                Arguments.of(new MockContinuation(Continuation.Reason.CURSOR_AFTER_LAST, new byte[]{2}, false, true)),
```

Reply:
Sure, Done.